src/en/page-templates/basic/basic.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Basic page template
+layout: 'layouts/page-templates-overview.njk'
+eleventyNavigation:
+  key: basicPageTemplateEN
+  title: Basic
+  locale: en
+  parent: pageTemplatesEN
+  description: Starting point for any web page.
+  descriptionSecondaryText: Offers a basic, accessible page structure and hierarchy and includes the elements required for most GC pages.
+  state: published
+#  tag: Canada.ca required
+  order: 1
+translationKey: 'basicPageTemplate'
+tags: ["templates"]
+date: "git Last Modified"
+github: https://github.com/cds-snc/gcds-examples/blob/feat/add-basic-page-templates/templates/english/basic-page-template.html
+figma: https://figma.com
+---
+
+# Basic page template
+
+The Basic Page template provides accessibility essentials for page structure and heading hierarchy. Brand elements, like the header and footer, make it recognizable as a Government of Canada page.
+
+## When to use
+
+For use on Canada.ca and other GC sites.
+The basic page template is for non-specialized pages across Canada.ca and is an optional template. It has the starting elements needed for a simple page and those required for most GC pages. You can use it as a canvas to add or combine <gcds-link href="{{ links.components }}">components</gcds-link> from the GC Design System.
+
+## Preview
+
+<gcds-button class="md:d-inline-block d-block md:me-400 me-0 md:mb-0 mb-400" button-role="secondary" type="link" href="{{ links.pageTemplatesBasicPreview }}" target="_blank">Open demo in new tab</gcds-button>
+<gcds-button class="md:d-inline-block d-block" button-role="secondary" type="link" href="{{ links.pageTemplatesBasicCode }}" target="_blank">Get code in new tab</gcds-button>
+
+<img class="max-width-content b-sm b-default mb-400 p-400" src="/images/en/templates/basic-page-preview.png" alt="A basic page template includes the Government of Canada header and footer, H1, H2 and H3 heading sections with text underneath each section."/>
```

Comment:
`mb-400` and `p-400` need to be changed to 300 please to match the new spacing classes.

Reply:
Done!